ldk/node/src/hostClients/networkClient.ts

```diff
@@ -64,7 +64,7 @@ export class NetworkClient
       },
       (response: messages.HTTPResponseMsg) => ({
         statusCode: response.getResponsecode(),
-        data: response.getData(),
+        data: response.getData_asU8(),
```

Comment:
@richseviora @BillyMurrayOlive - this is consistent with how we've done this in each LDK. Is there a reason we chose to do it this way instead of a decoded string? Do we expect that most partners won't typically decode the body anyways?

Reply:
Just chatted with Rich and I forgot that sometimes files are NOT just text 😀 Just makes sense